react-toggle-switch-component/src/toggle-switch.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+export default class ToggleSwtich extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isClicked: false
+    };
+
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+    this.setState({ isClicked: !this.state.isClicked });
+  }
+
+  render() {
+
+    const isClicked = this.state.isClicked;
+
+    if (isClicked) {
+      return (
+        <div className="outer" onClick={this.handleClick}>
+        <div className="inner"><p>OFF</p></div>
+      </div>
+      );
+    }
+    return (
+      <div className="outer click-one" onClick={this.handleClick}>
+      <div className="inner click-two"><p>ON</p></div>
+    </div>
+    );
```

Comment:
This will be hard to maintain and can lead to bugs and inconsistencies. A better approach would be:
```JSX
const isClicked = this.state.isClicked;
const outerClass = '';
const innerClass = '';
const text = 'OFF';

if (!isClicked) {
  outerClass = 'click-one';
  innerClass = 'click-two';
  text = 'ON';
}

return (
  <div className={`outer ${outerClass}`} onClick={this.handleClick}>
    <div className={`inner ${innerClass}`}>
      <p>{text}</p>
    </div>
  </div>
);
```